src/test/java/uk/nhs/hee/trainee/details/api/ProgrammeMembershipResourceTest.java

```diff
@@ -286,6 +288,26 @@ void getShouldReturnBadRequestWhenTokenNotFound() throws Exception {
         .andExpect(status().isBadRequest());
   }
 
+  @Test
+  void getShouldReturnBadRequestWhenTokenNotMap() throws Exception {
```

Comment:
Cool, I see this brings our token tests here into line with the tests we have in the actions and notifications services.